.gitignore

```diff
@@ -0,0 +1,3 @@
+/scripts/google-sheets/.clasp.json
+.idea/
+.DS_Store
```

Comment:
can you please move `.idea` and `.DS_store` to your global gitignore?
Reasoning: each developer has a different development environment. 
Since there are many, we don't want to add `.gitignore` settings for all of IDEs in all projects